### PR TITLE
New HUD Graphic

### PR DESCRIPTION
**Pull request by AnthonyWharton**
To be accepted shortly, but the start of a new HUD graphic is on this branch.

### CODE REVIEW

File: src/client/include/client/system/SystemRenderGUI.hpp [hunk 1]
```diff
@@ -4,15 +4,68 @@
...
-	            tempo::Clock &             clock,
-	            int                        combo,
-						  tempo::ComponentHealth     comp_health);
+
+	private:
+		// Misc Colours
+		irr::video::SColor colour_white = irr::video::SColor(255, 255, 255, 255);
+		irr::video::SColor colour_red   = irr::video::SColor(255, 255, 0, 0);
+		irr::video::SColor colour_green = irr::video::SColor(255, 0, 255, 0);
+		irr::video::SColor colour_blue  = irr::video::SColor(255, 135, 206, 250);
```
Comment (RcColes) [inline src/client/include/client/system/SystemRenderGUI.hpp]: We should look into these at some point - the pure colours are not that good looking

File: src/client/main.cpp [hunk 2]
```diff
@@ -150,7 +150,7 @@ int main(int argc, const char **argv)
 
 
 	irr::IrrlichtDevice *device = irr::createDevice(
-	  irr::video::EDT_OPENGL, irr::core::dimension2d<irr::u32>(1280, 720), 16, false, false, false);
+	  irr::video::EDT_OPENGL, irr::core::dimension2d<irr::u32>(1920, 1080), 16, true, false, false);
```
Comment (RcColes) [inline src/client/main.cpp]: This crashes my window manager (assuming it's fullscreen like I think it is), so could we change this to be some sort of command line flag to enable it, and then just add that to the lab machine init script?

File: src/client/include/client/system/SystemRenderGUI.hpp (hunk 1 above)
Comment (RcColes) [reply src/client/include/client/system/SystemRenderGUI.hpp]: That is assuming they're being used as is

File: src/client/main.cpp (hunk 2 above)
Comment (grant-m-s) [reply src/client/main.cpp]: yeah this for fullscreen

Comment (AnthonyWharton) [reply src/client/main.cpp]: Yeah it's 1080 fullscreen.. I had a feeling this might break your quite frankly __ludicrous__ setup 😆

Comment (RcColes) [reply src/client/main.cpp]: Yeah, because I previously mentioned it when we last tested it

**Comment by AnthonyWharton** [comment]
@Codrin-Popa I can't review changes as I made the PR.. but you've used spaces instead of hard tabs. Please convert the files you have dealt with the the standard team format. Give me a shout if you want a hand with that.

**Comment by Codrin-Popa** [comment]
Now the 3rd command line argument specifies if the hud should be enabled.
To enable the hud just write HUD or hud.

**Comment by grant-m-s** [comment]
bit strange to write hud for fullscreen but oh well

**Comment by Codrin-Popa** [comment]
I am going to add the login screen to this pull request as well

**Pull request merged**